sw/ground_segment/cockpit/live.ml

```diff
@@ -457,6 +461,10 @@ let create_ac = fun ?(confirm_kill=true) alert (geomap:G.widget) (acs_notebook:G
   let via_http = Str.string_match (Str.regexp "http") af_url 0 in
   let af_xml = ExtXml.parse_file ~noprovedtd:via_http af_file in
 
+  (** Get firmware name *)
+  let firmware = ExtXml.child af_xml "firmware" in
```

Comment:
Maybe this part should have been in another pull request then. Is a NO_WIND option in link also an option? Is no rotorcraft ever using wind?